apps/web/components/eventtype/RequiresConfirmationController.tsx

```diff
@@ -121,73 +128,86 @@ export default function RequiresConfirmationController({
                     )}
                     {(requiresConfirmationSetup !== undefined ||
                       !requiresConfirmationLockedProps.disabled) && (
-                      <RadioField
-                        disabled={requiresConfirmationLockedProps.disabled}
-                        className="items-center"
-                        label={
-                          <>
-                            <Trans
-                              i18nKey="when_booked_with_less_than_notice"
-                              defaults="When booked with less than <time></time> notice"
-                              components={{
-                                time: (
-                                  <div className="mx-2 inline-flex">
-                                    <Input
-                                      type="number"
-                                      min={1}
-                                      disabled={requiresConfirmationLockedProps.disabled}
-                                      onChange={(evt) => {
-                                        const val = Number(evt.target?.value);
-                                        setRequiresConfirmationSetup({
-                                          unit:
-                                            requiresConfirmationSetup?.unit ??
-                                            defaultRequiresConfirmationSetup.unit,
-                                          time: val,
-                                        });
-                                        formMethods.setValue(
-                                          "metadata.requiresConfirmationThreshold.time",
-                                          val,
-                                          { shouldDirty: true }
-                                        );
-                                      }}
-                                      className="border-default !m-0 block w-16 rounded-r-none border-r-0 text-sm [appearance:textfield] focus:z-10 focus:border-r"
-                                      defaultValue={metadata?.requiresConfirmationThreshold?.time || 30}
-                                    />
-                                    <label
-                                      className={classNames(
-                                        requiresConfirmationLockedProps.disabled && "cursor-not-allowed"
-                                      )}>
-                                      <Select
-                                        inputId="notice"
-                                        options={options}
-                                        isSearchable={false}
-                                        isDisabled={requiresConfirmationLockedProps.disabled}
-                                        innerClassNames={{ control: "rounded-l-none bg-subtle" }}
-                                        onChange={(opt) => {
+                      <>
+                        <RadioField
```

Comment:
This was moved into a fragment so theres a big diff here. All that was changed was there was a checkbox added to the bottom of the radioField